partner_identification_dea/models/res_partner.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2021 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    dea_number = fields.Char(
+        string="DEA #",
+    )
+    dea_expired_date = fields.Date(string="DEA Expiration Date")
+    dea_active = fields.Selection([("yes", "Yes"), ("no", "NO")], string="DEA Active")
+
+    medical_license = fields.Char(
+        string="Medical License",
+    )
+    medical_license_expired_date = fields.Date(string="Medical License Expiration Date")
```

Comment:
@Murtaza-OSI All above information already exist on **res.partner.id_category**. You only need to put many2one field to res.partner.id_category from res.partner. For example https://github.com/open-synergy/opnsynid-vertical-accountant/blob/8.0/partner_identification_cpa_license/models/res_partner.py

Then you have to add res.partner.id_category data. For example https://github.com/open-synergy/opnsynid-vertical-accountant/blob/8.0/partner_identification_cpa_license/data/res_partner_id_category_data.xml